CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Changelog
 
-### Unreleased
+## Unreleased
 
 - The client has been adopted to work with Exonum dynamic_services branch.
-- Proto files are now downloaded via REST API and compiled dynamically.
+- Proto files are now downloaded via REST API from the Exonum node
+and compiled dynamically.
```

Comment:
Linter doesn't pass this line. 
```suggestion
  and compiled dynamically.
```

Reply:
Strange, vs code markdown linter didn't show that one :(

Reply:
At least a linter of the main `exonum` repo 🙃. It would be good to have the same format, wouldn't it ?

Reply:
Totally agree.